cfgov/unprocessed/js/routes/common.js

```diff
@@ -21,7 +21,6 @@ require( '../modules/form-validation' ).init();
 require( '../modules/reveal-on-focus' ).init();
 require( '../modules/scroll-on-history-collapse' ).init();
 require( '../modules/clear-form-buttons' ).init();
-require( '../modules/jquery/custom-input' ).init();
```

Comment:
We resolved this in chat, but the last buttons are in the filters that are being re-done in #1185. I'll fix those before merging, but I'm waiting until that's merged and resolved.